ultranest/popstepsampler.py

```diff
@@ -536,8 +536,219 @@ def __next__(
         else:
             return None, None, None, nc
 
+class PopulationSimpleSliceSampler():
+    """
+       Vectorized Slice sampler without stepping out procedure.
+       In Comparison the PopulationSliceSampler, the sampler calls the likelihood on
+       batch of points of the same size.
+
+       Sliced are defined by the generate_direction function on a interval defined
+       around the current point. The centred interval has the width of the scale parameter.
+       Slices are then shrink towards the current point until a point is found with a
+       likelihood above the threshold.
+
+       A slice can be searched with more than one point at a time. In that case, we 
```

Comment:
Multiple (`popsize`) slice sampling chains are run independently and in parallel.

Reply:
I added your line